SingelLinkedList/LinkedList.cs

```diff
@@ -77,6 +77,29 @@ public Node GetNode(int argData)
             }
             return currentNode;
         }
+        public Node SwitchNodes(int firstNode, int secondNode)
+        {
+            Node _firstNode = head;
+            Node _secondNode = head;
+            while (_firstNode?.data != firstNode & _secondNode?.next.data != secondNode)
```

Comment:
Eigentlich eine coole Idee beides in das Abbruchkriterium zu packen... was passiert hier, wenn das nie erfüllt ist?